cudaaligner/include/claragenomics/cudaaligner/alignment.hpp

```diff
@@ -24,11 +24,23 @@ namespace cudaaligner
 /// \addtogroup cudaaligner
 /// \{
 
-/// \typedef FormattedAlignment
-/// Holds formatted strings representing an alignment.
-/// FormattedAlignment.first = formatted string for query
-/// FormattedAlignment.second = formatted string for target
-typedef std::pair<std::string, std::string> FormattedAlignment;
+/// FormattedAlignment -Holds formatted strings representing an alignment.
+typedef struct FormattedAlignment
+{
+    /// \brief FormattedAlignment.query = formatted string for query
+    std::string query;
+    /// \brief FormattedAlignment.pairing = formatted pairing string
+    std::string pairing;
+    /// \brief FormattedAlignment.target = formatted string for target
+    std::string target;
+    /// \brief Maximal line length when outputting to ostream (default = 0, no line break)
+    std::size_t linebreak_after = 0;
```

Comment:
I would make this an `int32_t`, since it is a count and will never need to be so big that it can address every byte of memory. This would also match the argument of the `format_alignment(...)` method then.